packages/core/src/App/Containers/RealAccountSignup/address-details-form.js

```diff
@@ -80,7 +80,7 @@ export const addressDetailsConfig = ({ can_upgrade_to, residence }) => {
  * @return {object} rules - Transformed rules
  */
 const transformForResidence = (rules, residence) => {
-    // IM Clients does not need to fillout state since API states_list is empty.
+    // IM Clients does not need to fill out state since API states_list is empty.
     if (residence === 'im') {
```

Comment:
im or iom?

Reply:
`im` is returned from the API and it stands for `Isle of Man` residence. But not a bad idea to update the comment. Thanks for pointing it out.